tests/excerptexport/test_conversion_api_client.py

```diff
@@ -17,222 +16,249 @@
+    mocker.patch.object(
         ConversionApiClient, 'create_parametrization',
         side_effect=[sentinel.parametrization_1, sentinel.parametrization_2],
     )
-    @mock.patch.object(ConversionApiClient, 'create_boundary')
-    def test_extraction_order_forward_to_conversion_service(
-            self, create_boundary_mock, create_parametrization_mock, create_job_mock):
-        result = self.extraction_order.forward_to_conversion_service(incoming_request=self.request)
+    mocker.patch.object(ConversionApiClient, 'create_boundary')
 
-        create_boundary_mock.assert_called_once_with(self.bounding_box.geometry, name=self.excerpt.name)
-        gis_conversion_options = self.extraction_order.extraction_configuration['gis_options']
-        assert_that(
-            create_parametrization_mock.mock_calls, contains_inanyorder(
-                mock.call(create_boundary_mock.return_value, 'fgdb', gis_conversion_options),
-                mock.call(create_boundary_mock.return_value, 'spatialite', gis_conversion_options),
-            )
+    result = extraction_order.forward_to_conversion_service(incoming_request=job_progress_request)
+
+    ConversionApiClient.create_boundary.assert_called_once_with(bounding_box.geometry, name=excerpt.name)
+    gis_conversion_options = extraction_order.extraction_configuration['gis_options']
+    assert_that(
+        ConversionApiClient.create_parametrization.mock_calls, contains_inanyorder(
+            mock.call(ConversionApiClient.create_boundary.return_value, 'fgdb', gis_conversion_options),
+            mock.call(ConversionApiClient.create_boundary.return_value, 'spatialite', gis_conversion_options),
         )
-        assert_that(
-            create_job_mock.mock_calls, contains_inanyorder(
-                mock.call(sentinel.parametrization_1, self.request),
-                mock.call(sentinel.parametrization_2, self.request),
-            )
+    )
+    assert_that(
+        ConversionApiClient.create_job.mock_calls, contains_inanyorder(
+            mock.call(sentinel.parametrization_1, job_progress_request),
+            mock.call(sentinel.parametrization_2, job_progress_request),
         )
-        assert_that(
-            result, contains_inanyorder(
-                sentinel.job_1,
-                sentinel.job_2,
-            )
+    )
+    assert_that(
+        result, contains_inanyorder(
+            sentinel.job_1,
+            sentinel.job_2,
         )
+    )
+
 
-    #
-    # Integration tests:
+@pytest.fixture
+def api_client():
+    return ConversionApiClient()
 
-    # TODO: re-record VCR (service API has changed)
 
-    @vcr.use_cassette('fixtures/vcr/conversion_api-test_create_job.yml')
-    def test_create_job(self):
-        self.assertIsNone(self.extraction_order.process_id)
+#
+# ConversionApiClient integration tests:
 
-        response = self.api_client.create_job(self.extraction_order, request=self.request)
+# TODO: re-record VCR (service API has changed)
```

Comment:
Actually, this comment was here [before this PR](https://github.com/geometalab/osmaxx-frontend/pull/426/files/d61c77df4cf6d7ac58132747b2dac9fc0c957964#diff-b7bf2d14794ca379596eebbdc057b222L109) already.